styles/styles.css

```diff
@@ -533,6 +535,19 @@ body.home > main a {
   text-align: center;
 }
 
+body.home h3 {
+  font-size: var(--heading-font-size-l);
```

Comment:
please change to `--font-size: ...`, so it can be used for the relative size calculation here: https://github.com/hlxsites/elco-aerotop-sg/blob/80d6ef9c4a8e34e5c14ea9dfafecbfa468e8be4b/styles/styles.css#L210

Reply:
done